omnibus/config/software/oc_id.rb

```diff
@@ -28,6 +28,18 @@
 build do
   env = with_standard_compiler_flags(with_embedded_path)
 
+  # Tell nokogiri to use the system libraries instead of compiling its own
+  env["NOKOGIRI_USE_SYSTEM_LIBRARIES"] = "true"
```

Comment:
Why do we need both this environment variable and the `--use-system-libraries` flag in the bundle command below?  The documentation only mentions the flag, so it might be good to mention why we need this in addition to the flag.